### ISSUE TITLE
[Feature Request] Allow display of expandable arrow icon when "expand" prop used in data tables. 

### ISSUE THREAD

**Issue by SnakeyHips** [vuetifyjs/vuetify#5495]
### Problem to solve
There is currently no visual indication to the user if a data table's rows are expandable or not which results in either the user missing out on the feature or them having to be informed about it. 

### Proposed solution
Allow the ability to append the expandable arrow icon with animation, as used for the expansion panel component, for data table rows if the "expand" prop is used i.e. as an "expand-icons" prop.

**Comment by gwh-cpnet** [comment]
```
<td>
  <v-btn small flat icon @click="props.expanded = !props.expanded">
    <v-icon small v-if="props.expanded">mdi-chevron-down</v-icon>
    <v-icon small v-else>mdi-chevron-right</v-icon>
  </v-btn>
</td>
```

may just solve the problem

I have my another issue #7172 about this. What I want to figure out is how to toggle the expansion somewhere else, where I can not pass `props.expanded` into.

**Comment by nekosaur** [comment]
This is included in 2.0 rewrite of data-table.
